packages/apollo-server-core/src/ApolloServer.ts

```diff
@@ -266,13 +271,13 @@ export class ApolloServerBase<Request = RequestInit> {
 `);
       }
 
-      this.engine = new ApolloEngine(
-        typeof engine === 'boolean' ? undefined : engine,
-      );
+      this.engineProxy = new ApolloEngine(engineProxy);
+      // XXX should this allow for header overrides from graphql-playground?
+      if (this.engineProxy) this.engineEnabled = true;
+    } else {
+      //engine is already in the request path
```

Comment:
I'm not sure that `engineProxy: true` really suggests "it's already running" to me. I think a separate option like we used to have is fine, or maybe a special string value like `engineProxy: 'external'` or something?